pages/linux/chage.md

```diff
@@ -0,0 +1,24 @@
+# chage
+
+> Change user account and password expiry information.
+
+- List password information for the user:
+
+`chage -l {{user_name}}`
+
+- Enable password expiration:
```

Comment:
I think you can combine this and the next example with - 
```
Set the number of days to be 10, after which the user has to change the password

sudo chage -m {{10}} {{user_name}}
```

Reply:
could you please check the fixed version?